es6/spec/database/mock.where.spec.js

```diff
@@ -0,0 +1,37 @@
+import Database from "../../lib/database.js";
+
+const databaseConfig = require("../../../database.json").testing;
+
+describe(".where", () => {
+	let database,
+		mockUsers;
+
+	beforeEach(() => {
+		database = new Database(databaseConfig);
+		mockUsers = [
+			{id: 1, name: "Bob"}
+		];
+	});
+
+	it("should treat where equals to andWhere", done => {
+		const name = "aName";
+
+		database.mock.select("*").from("users").where("created_at", ">", /.*/).where("name", name).results(mockUsers);
+
+		database.select("*").from("users").where("created_at", ">", "2015-10-02 21:39:14").andWhere("name", name).results((error, rows) => {
+			if (error) { throw error; }
+			rows.should.eql(mockUsers);
+			done();
+		});
+	});
+
+	it("should use equals as the default operator", done => {
```

Comment:
no, this is testing that it works with equals as default, while the other is testing that andWhere and where are treated as equals